hack/make-rules/test.sh

```diff
@@ -211,6 +211,41 @@ junitFilenamePrefix() {
   echo "${KUBE_JUNIT_REPORT_DIR}/junit_${KUBE_TEST_API_HASH}_$(kube::util::sortable_date)"
 }
 
+verifyAndSuggestPackagePath() {
+  local specified_package_path="$1"
+  local alternative_package_path="$2"
+  local original_package_path="$3"
+  local suggestion_package_path="$4"
+
+  if ! [ -d "$specified_package_path" ]; then
+    # Because k8s sets a localized $GOPATH for testing, seeing the actual
+    # directory can be confusing. Instead, just show $GOPATH if it exists in the
+    # $specified_package_path.
+    printable_package_path=$(echo "$specified_package_path" | sed "s|$GOPATH|\$GOPATH|")
```

Comment:
should be local var

Reply:
Great catch @cblecker ! Made fix and updated diff.